bench/irmin-pack/trace_replay_intf.ml

```diff
@@ -13,19 +13,54 @@
  * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
-type config = {
-  ncommits_trace : int;
-  store_dir : string;
-  path_conversion : [ `None | `V1 | `V0_and_v1 | `V0 ];
-  inode_config : int * int;
-  store_type : [ `Pack | `Pack_layered | `Pack_mem ];
-  commit_data_file : string;
-  artefacts_dir : string;
-  keep_store : bool;
-  keep_stat_trace : bool;
-  no_summary : bool;
-  empty_blobs : bool;
-}
+
+module Config = struct
+  type config = {
+    number_of_commits_to_replay : int;
+    path_conversion : [ `None | `V1 | `V0_and_v1 | `V0 ];
+    inode_config : int * int;
+    store_type : [ `Pack | `Pack_layered | `Pack_mem ];
+    path_to_trace_to_replay : string;
+    path_to_artefacts : string;
+    keep_store : bool;
+    keep_stat_trace : bool;
+    empty_blobs : bool;
+  }
+  (** Replay configuration
+
+      [path_to_trace_to_replay] points to a specific file that describes the
+      sequence of operations to replay. You may download one of the following
+      URLs. The smaller ones are prefix of the larger ones.
+
+      - http://data.tarides.com/irmin/data4_10310commits.repr (0.3GB)
+      - http://data.tarides.com/irmin/data4_100066commits.repr (2.9GB)
+      - http://data.tarides.com/irmin/data_1343496commits.repr (102GB)
+
+      [number_of_commits_to_replay] is the wished number of commits to replay.
+      If the value is too high, the replay will stop when reaching the end of
+      [path_to_trace_to_replay]. Pick a number of commits depending on the
+      wished runtime. Here are some reference runtimes that were true for irmin
+      3.0:
+
+      - [60_457] commits take 3 minutes
+      - [500_000] commits take 1 hour
+      - [1_343_496] commits take 5 hours
+
+      [path_to_artefacts] is the desination for the stats trace and the store.
+      If both [keep_store] and [keep_stat_trace] are false, the destination will
+      be emptied at the end of the replay.
+
+      [path_conversion] interprets the paths read in the trace differently. Some
+      specific nesting levels are removed, which speeds up the replay.
```

Comment:
IMHO, either change the name or change the message. I feel like it's a bit confusing for users....

Reply:
`path_conversion ` is really a legacy thing, it could be removed. I will annotate it with `Do not use unless you know what you are doing`